### ISSUE TITLE
Request - Add Firefox profile settings

### ISSUE THREAD

**Issue by patgmac** [erikberglund/ProfileCreator#92]
Firefox 64 can now use configuration profiles. Would be awesome to see it be added to ProfileCreator. 

https://github.com/mozilla/policy-templates/tree/master/mac

**Comment by erikberglund** [comment]
Agreed, that would be great. If someone want to give it a go please do so, write here if you do so we dont do double work. I will get to this in maybe a week as i have a brunch of things to do before i can do this

**Comment by patgmac** [comment]
Not volunteering yet, since it's probably beyond my current capabilities. But where's the source code? The repo only has a readme and a screen shot.

**Comment by kevinmcox** [comment]
Those are created in the ProfileManifests repo: https://github.com/erikberglund/ProfileManifests

**Comment by erikberglund** [comment]
In a recent version I added two cool features that will make creating manifests for plist files super easy. The first was to read in generic plists and present in the app under the little computer icon, doing basically what mcxToProfile does. The second was a developer option to show how the current manifest it's using for the GUI actually looks in XML.

If I combine these two features, I have just created the base for a manifest for firefox in ~20 seconds.

I will look more closely after I fixed my last item on the list for release, it's giving me a hard time. Then I will add this payload.


[org.mozilla.firefox.txt](https://github.com/erikberglund/ProfileCreator/files/2679119/org.mozilla.firefox.txt)
